### PR TITLE
Allow mounting any partition of a complete disk image + test.

### PR DESCRIPTION
**Pull request by zvin**

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -2,33 +2,88 @@ const lkl = require('../');
 const fs = require('fs');
 const path = require('path');
 const assert = require('assert');
+const async = require('async');
```
Comment (petrosagg) [inline test/index.js]: Module missing from package.json

File: test/index.js [hunk 2]
```diff
@@ -2,33 +2,88 @@ const lkl = require('../');
...
+
+		// create a copy of the ext4 partition
 		fs.createReadStream(__dirname + '/fixtures/test.ext4')
 		.pipe(fs.createWriteStream(this.disk))
-		.on('close', done);
+		.on('close', function() {
+			// then create a copy of the partitionned disk.
+			// The disk image is compressed in order to avoid github's file
+			// size restrictions.
+			fs.createReadStream(__dirname + '/fixtures/disk.img.gz')
```
Comment (petrosagg) [inline test/index.js]: The uncompressed size is 343MB. Why not something much smaller like the 4MB ext4 partition?

Comment (zvin) [inline test/index.js]: I wanted to test that we can mount any file system.
All the partitions have the minimum required size for each FS:


| number | file system | size   |
|--------|-------------|--------|
| 1      | btrfs       | 256Mio |
| 2      | ext2        | 1Mio   |
| 3      | ext4        | 1Mio   |
| 4      | extended    |        |
| 5      | fat16       | 16Mio  |
| 6      | fat32       | 33Mio  |
| 7      | xfs         | 32Mio  |

This totals to 339Mio ~= 355MB

The compressed image is less than 427Kib and the decompression takes less than 2 seconds so I didn't think it was a problem.

We could remove the btrfs one and save 256 uncompressed Mio.

File: test/index.js (hunk 1 above)
Comment (zvin) [inline test/index.js]: fixed

File: test/index.js (hunk 2 above)
Comment (petrosagg) [inline test/index.js]: Oh ok. I didn't know btrfs had such a high minimum size

**Comment by petrosagg** [comment]
The ext4 fixture looks modified but I don't see something in the test code that depends on it. Accidental inclusion?

**Comment by zvin** [comment]
Yes it was an error, I restored the original ext4 image.

**Comment by petrosagg** [comment]
@zvin I just realised that the disk.img.gz was gone together with the ext4 image. I'll push to master

**Comment by zvin** [comment]
@petrosagg sorry I forgot to add it again when I updated my commit to remove test.ext4 changes.

**Pull request merged**